data/notices.json

```diff
@@ -79,6 +79,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(lambda-layer-awscli): [ACTION REQUIRED] Your CDK application is using AwsCliLayer",
+      "issueNumber": 22470,
+      "overview": "Your app is using the AwsCliLayer construct in an environment where npm is inaccessible. Add a dependency on @aws-cdk/asset-awscli-v1, or the equivalent in your language, to remove this notice.",
+      "components": [
+        {
+          "name": "aws-cdk-lib.lambda_layer_awscli.Notice",
+          "version": ">=2.x.y"
```

Comment:
Interesting. This seems to be a use case where we don't want an upper bound on the version. Is this correct?

Reply:
Yes, that's the use case. Your question made me realize I should just be using the `^` notation. That works.

Reply:
I'm now using a placeholder, which causes the build to fail, until we know what version of aws-cdk-lib the relevant change is going out in.